pkg/api/transform.go

```diff
@@ -1,36 +1,20 @@
 package api
 
 import (
-	"strings"
-
-	"github.com/treeverse/lakefs/pkg/api/gen/models"
 	"github.com/treeverse/lakefs/pkg/catalog"
 )
 
 func transformDifferenceTypeToString(d catalog.DifferenceType) string {
 	switch d {
 	case catalog.DifferenceTypeAdded:
-		return models.DiffTypeAdded
+		return "added"
```

Comment:
An advantage of doing it the old way is that you get compile-time checking.  If oapi gives named enum constants for the allowed values, it would be better to continue to use them here.

(Also below, e.g. "allow"/"deny")

Reply:
Agree I think this is another reason to fork or suggest a PR to have the code generation introduce these constants.
From what I saw it generates constants just for top-level enums.